build/packer/ansible/roles/podman-setup/templates/pmm-server.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=pmm-server
+Wants=network-online.target
+After=network-online.target
+After=nss-user-lookup.target nss-lookup.target
+After=time-sync.target
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+
+ExecStart=/usr/bin/podman run --volume /srv/:/srv/ --rm --name %N \
+    -p 8443:8443/tcp --ulimit=host {{ pmm_server_image_name }}
```

Comment:
Why don't we map the container port 8443 to the host port 443? It seems to be so much more straightforward.